spec/src/main/asciidoc/microprofile-opentracing.asciidoc

```diff
@@ -135,8 +135,8 @@ The implementation has to provide two operation name providers:
 * `class-method` - the provider for the default operation name.
 * `http-path` - the operation name has the following form
 `<HTTP method>:<@Path value of endpoint's class>/<@Path value of endpoint's method>`.
-For example if the class is annotated with `@Path(service)` and method `@Path(endpoint)` then
-the operation name is `GET:/service/endpoint`.
+For example if the class is annotated with `@Path("service")` and method `@Path("endpoint\{id: \\d+}")` then
```

Comment:
Should `@Path("endpoint\{id: \\d+}")` be `@Path("endpoint/{id: \\d+}")`?

Reply:
yes there is a typo. I will fix it